evm/src/cpu/kernel/asm/mpt/hex_prefix.asm

```diff
@@ -5,100 +5,124 @@
 //
 // Pre stack: rlp_start_pos, num_nibbles, packed_nibbles, terminated, retdest
 // Post stack: rlp_end_pos
-
 global hex_prefix_rlp:
-    // stack: rlp_pos, num_nibbles, packed_nibbles, terminated, retdest
-    // We will iterate backwards, from i = num_nibbles / 2 to i = 0, so that we
-    // can take nibbles from the least-significant end of packed_nibbles.
-    PUSH 2 DUP3 DIV // i = num_nibbles / 2
-    // stack: i, rlp_pos, num_nibbles, packed_nibbles, terminated, retdest
-
+    DUP2 %assert_lt_const(65) // TODO: The number of nibbles can't be more than 64?
```

Comment:
Do we have a ticket for this `TODO`? If yes, would be good to add a ref to it here.

Reply:
Idk, I think 64 means there can't be more than 2²⁵⁶ accounts (leafs in the state trie). Don't know if that's a problem  or not.

Reply:
There can't be more than 2^256 addresses, so the TODO can be removed.